internal/controller/observability/collector.go

```diff
@@ -29,6 +29,23 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+// ReconcileCollector adds scc, owner, options, context,
```

Comment:
Can we make this more succinct regarding the description.  The verbosity lends itself to skew when the function is updated but the comment is not. All the information here can be understood by reading the calls and if the subsequent calls do not clearly identify intent then we should modify those function names.  I would say this comment should be something like:

```
ReconcileCollector generates and deploys the resources (i.e. ConfigMap, Secrets, Deployments, etc) to realize the spec of the ClusterLogForwarder
```

Reply:
Yep, I didn't mean to leave this in.  I have a refactor that breaks up this monolith function.